sbol3/document.py

```diff
@@ -124,7 +123,7 @@ def _build_extension_object(self, identity: str, sbol_type: str,
                 build_type = type_uri
                 break
             except KeyError:
-                logging.warning(f'No builder found for {type_uri}')
+                logging.warning('No builder for %s', type_uri)
```

Comment:
We should maintain the text of the warning message unless there is a good reason to change it.
```suggestion
                logging.warning('No builder found for %s', type_uri)
```